app/static/tags/accountpanel.tag

```diff
@@ -0,0 +1,65 @@
+<div id="panel-account" class="col-md-12 col-lg-12" ng-controller="accountuserController">
+    <div class="panel panel-default">
+        <div class="panel-heading panel-heading-nav">
+            <a id="account-btn" data-target="#panel-account-collapse" class="btn btn-info btn-fa-2x" role="button" data-toggle="collapse"><i class="fa fa-angle-down fa-2x fa-fw-forced"></i></a>
+            <h4> account details</h4>
+        </div>
+
+        <div id="panel-account-collapse" class="panel-btn-in-header-collapse collapse">
+            <div class="panel-body">
+
+                <div class="panel-group" id="account-list-accordion">
+
+                    <!-- ............. ACCOUNT DETAILS ............................................... -->
+
+                        <form class="form-horizontal" role="document"><!-- role: 'document' - non-editable "form" -->
+                            <div class="form-group">
+                                <label class="col-sm-3 control-label" for="awsclusterName">Username</label>
+
+                                <div class="col-sm-9">
+                                    <p id="awsclusterName" class="form-control-static">{{userDetails.userName}}</p>
+                                </div>
+                                <!-- .col-sm-9 -->
+                            </div>
+                            <div class="form-group">
+                                <label class="col-sm-3 control-label" for="awsclusterName">First name</label>
+
+                                <div class="col-sm-9">
+                                    <p id="awsclusterName" class="form-control-static">{{userDetails.givenName}}</p>
```

Comment:
Please create unique id and name attribute for the different form groups that identify which field will be presented by the group. Please do this for the following groups too!